quipucords/scanner/satellite/inspect.py

```diff
@@ -40,8 +43,21 @@ def __init__(self, scan_job, scan_task):
         self.connect_scan_task = None
 
     # pylint: disable=too-many-return-statements
-    def run(self):
+    def run(self, manager_interrupt):
         """Scan satellite manager and obtain host facts."""
+        # Make sure job is not cancelled or paused
```

Comment:
Not a blocker, but since this is repetitive block of code, consider putting it in a util

Reply:
😶